database-mongodb/usersSeed.js

```diff
@@ -0,0 +1,113 @@
+// SEEDING SCRIPT
+// "db:usersSeed": "node database-mongodb/usersSeed.js"
+
+const faker = require('faker');
+const db = require('./index.js');
+const User = require('./user.js');
+
+const sampleUsers = [];
+
+function getRandomArbitrary(min, max) {
+  return Math.random() * (max - min) + min;
+}
+
+function getRandomGender() {
+  const zeroOrOne = Math.floor(getRandomArbitrary(0, 2));
+  if (zeroOrOne === 0) {
+    return 'Male';
+  }
+  return 'Female';
+}
+
+function getRandomAge18To45() {
+  return Math.floor(getRandomArbitrary(18, 45));
+}
+
+const travelExperience = {
+  0: 'Novice Nomad',
+  1: 'Avid Traveller',
+  2: 'Globetrotter',
+};
+
+function getRandomTravelerExperience() {
+  const travelerNum = Math.floor(getRandomArbitrary(0, 3));
+  return travelExperience[travelerNum];
+}
+
+function randomNumber(minimum, maximum, precision) {
+  const min = minimum === undefined ? 0 : minimum;
+  const max = maximum === undefined ? 9007199254740992 : maximum;
+  const precision1 = precision === undefined ? 0 : precision;
```

Comment:
Same as above

Reply:
Line 40,

Do you know why I can't use `precision` as a var name? There error I get is `Parsing error: Identifier 'precision' has already been declared`